MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include scripts *.txt
```

Comment:
this should be moved to the `pyproject.toml` file. Here: https://github.com/bioio-devs/bioio-imageio/blob/main/pyproject.toml#L77